docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,2916 @@
+
+[float]
+[[features-8.6.0]]
+=== Features
+{kib} 8.6.0 adds the following new and notable features.
+
+Alerting::
+* Notify users by email when assigned to a case {kibana-pull}144391[#144391]
+* Adds flapping state object and interface in AAD index and Event Log {kibana-pull}143920[#143920]
+* Change Alerts > Actions execution order {kibana-pull}143577[#143577]
+* Adds the ability to remove alerts attached to a case {kibana-pull}143457[#143457]
+* This feature allows users to create and close alerts within Opsgenie {kibana-pull}142411[#142411]
+* Adds filter field to index threshold rule type {kibana-pull}142255[#142255]
+* Allow users to see event logs from all spaces they have access to {kibana-pull}140449[#140449]
+
+Elastic Security::
+For the Elastic Security 8.6.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+
+Enterprise Search::
+For the Elastic Enterprise Search 8.6.0 release information, refer to {enterprise-search-ref}/changelog.html[_Elastic Enterprise Search Documentation Release notes_].
+
+Fleet::
+* Differentiate kubernetes integration multi page experience {kibana-pull}145224[#145224]
+* Prerelease toggle {kibana-pull}143853[#143853]
+* Adds link to skip multi step add integration workflow {kibana-pull}143279[#143279]
+* Request diagnostics {kibana-pull}142369[#142369]
+
+Infrastructure::
+Adds support for the the Unified Search Bar for Query input {kibana-pull}143222[#143222]
+
+Lens & Visualizations::
+Adds support for trend lines in *Lens* metric visualizations {kibana-pull}141851[#141851]
+
+Machine Learning::
+* Trained model testing with index data {kibana-pull}144629[#144629]
+* Adding anomaly score explanations {kibana-pull}142999[#142999]
+
+Management::
+Adds the ability to show sub-feature privileges when using the Basic license {kibana-pull}142020[#142020]
+
+Monitoring::
+Collect metrics about the active/idle connections to ES nodes {kibana-pull}141434[#141434]
+
+Observability::
+* Integrate alert search bar on rule details page {kibana-pull}144718[#144718]
+* Adds additional context to recovered alerts of Infrastructure rules {kibana-pull}144683[#144683]
+* Adds list of containers in context variable of Inventory rule {kibana-pull}144526[#144526]
+* Adds new contextual attributes to Infrastructure - Metric threshold rule {kibana-pull}143001[#143001]
+* Adds alert details page feature flag by App {kibana-pull}142839[#142839]
+* Adds new contextual attributes to Infrastructure - Inventory Rule {kibana-pull}140598[#140598]
+
+Platform::
+Adds notifications plugin, offering basic email service {kibana-pull}143303[#143303]
+
+Security::
+The API Keys screen can be accessed in a Read Only view with the the cluster privilege `read_security` {kibana-pull}144923[#144923]
```

Comment:
__note:__ Moved a feature from `Management` section above and re-labeled API keys related entry as enhancement.
```suggestion
Adds the ability to show sub-feature privileges when using the Basic license {kibana-pull}142020[#142020]
```